blip-project/src/components/Page/Signup/signup.jsx

```
@@ -0,0 +1,529 @@
+import { typography } from "../../../fonts/fonts.jsx";
+import * as S from "./signupStyle.jsx";
+import Input from "../SignUpLoginComponent/input.jsx";
+import { color } from "../../../style/color.jsx";
+import { PassWord } from "../SignUpLoginComponent/password.jsx";
+import { Email } from "../SignUpLoginComponent/email.jsx";
+import Id from "../SignUpLoginComponent/id.jsx";
+import { PasswordCheck } from "../SignUpLoginComponent/passwordCheck.jsx";
+import modalX from "../../../svg/modalX.svg";
+import axios from "axios";
+import sends from "../../../svg/send.svg";
+import Number from "../SignUpLoginComponent/number.jsx";
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)